integration-test/common/pom.xml

```diff
@@ -144,6 +144,7 @@
                         <useFile>${test.redirectToFile}</useFile>
                         <argLine>-ea -Xmx3g</argLine>
                         <failIfNoTests>false</failIfNoTests>
+                        <trimStackTrace>${trimStackTrace}</trimStackTrace>
```

Comment:
Why do we need this? Is it enough to change parent pom file? And why do we only modify common, mr, spark-common? How about spark2 and spark3?

Reply:
Tested. The trimStackTrace in parent pom is enough.